src/Button/Button.less

```diff
@@ -22,7 +22,7 @@
 
 a, button {
   &.Button {
-    .type_medium;
+    .text--medium;
```

Comment:
nice. yeah, just did a search and the only place where they're used in jsx are the user-search-ui repo i added recently. just 2 instances - i can update those one this is ready